### PR TITLE
Update delete-calendar.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to reduce ambiguity between users and booking calendar

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9155

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review.

File: microsoft-365/bookings/delete-calendar.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The booking calendar is where all relevant information about that booking calend
 
    ![Image of Users UI in Microsoft 365 admin center.](../media/bookings-admin-center-users.png)
 
-1. On the **Active Users** page, choose the names of the users that you want to delete, and then select **Delete user**.
+1. On the **Active Users** page, choose the names of the Booking calendar that you want to delete, and then select **Delete user**.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/delete-calendar.md]:
```suggestion
1. On the **Active Users** page, choose the name of the booking calendar that you want to delete and then select **Delete user**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**